pkg/procmatch/readme.md

```diff
@@ -0,0 +1,52 @@
+# procmatch
+
+Procmatch is a library that provides a way to extract datadog-integrations from command lines by using a catalog.
```

Comment:
I think in this documentation I'd like to see step-by-step process on:

1. Adding a new integration + signatures
2. Adding tests for an integration + signature set

I think what you have already covers some of this but it'd be worth stating these use-cases clearly. I'm also not convinced that we would have a case where someone just installs the procmatch package/command separately.